### PR TITLE
Update Single DC Multi K8S doc

### PR DESCRIPTION
**Pull request by Ranjandas**
### Description
Updated the Single DC Multi K8S doc to have the following details:

* Reference to supported network offerings of various hosted K8S providers
* Added a command to explicitly switch the kubectl context.

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [ ] not a security concern
* [ ] checklist [folder](./../docs/config) consulted

### CODE REVIEW

File: website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx [hunk 1]
```diff
@@ -96,6 +98,10 @@ $ kubectl get secret consul-gossip-encryption-key ${HELM_RELEASE_SERVER}-consul-
 Now we can switch to the second Kubernetes cluster where we will deploy only the Consul clients
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx:98]:
```suggestion
Switch to the second Kubernetes cluster where you will deploy the Consul clients
```
Every instance of "we" should either be "you" or revised into an imperative statement with an implied "you". Per our style guide, use "we" when referring to HashiCorp. We do not use the plural first person (the royal we).

File: website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx [hunk 2]
```diff
@@ -96,6 +98,10 @@ $ kubectl get secret consul-gossip-encryption-key ${HELM_RELEASE_SERVER}-consul-
...
 that will join the first Consul cluster.
 
+```shell-session
+$ kubectl config use-context <CONTEXT_NAME>
+```
+
 First, we need to apply credentials we've extracted from the first cluster to the second cluster:
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx:105]:
```suggestion
First, apply the credentials you extracted from the first cluster to the second cluster:
```

**Review by trujillo-adam — CHANGES_REQUESTED** [review]
We should create a Requirements section for the requirements specified in the beginning of this topic. Please change instances of "we" to "you" or revise so that "you" is implied.

File: website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx (hunk 1 above)
Comment (Ranjandas) [reply website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx]: Thank you, @trujillo-adam, for the review. I will incorporate the changes you have proposed.

File: website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx [hunk 3]
```diff
@@ -14,7 +14,9 @@ In this example, we will use two Kubernetes clusters, but this approach could be
 
 ~> **Note:** This deployment topology requires that your Kubernetes clusters have a flat network
 for both pods and nodes, so that pods or nodes from one cluster can connect
-to pods or nodes in another. If a flat network is not available across all Kubernetes clusters, follow the instructions for using [Admin Partitions](/docs/enterprise/admin-partitions), which is a Consul Enterprise feature. 
+to pods or nodes in another. In many hosted Kubernetes environments, you will need to explicitly configure your hosting providers network. See [Azure AKS CNI](https://docs.microsoft.com/en-us/azure/aks/concepts-network#azure-cni-advanced-networking), [AWS EKS CNI](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) and [GKE VPC-native clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips).
+If a flat network is not available across all Kubernetes clusters, follow the instructions for using [Admin Partitions](/docs/enterprise/admin-partitions), which is a Consul Enterprise feature.
+
```
[the remark this answers is not in the document]
Comment (Ranjandas) [reply website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx:22]: I will add a new requirements section.

**Pull request merged**